src/app/components/CommentList.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import axios from 'axios';
+
+import Comment from '../components/Comment';
+
+export default class CommentList extends React.Component {
+
+    constructor (props) {
+        super(props);
+
+        this.state = {
+            comment: [],
+        }
+    }
+
+    render() {
+
+        let comment = <div>Идет загрузка...</div>;
```

Comment:
```suggestion
        let comments = <div>Идет загрузка...</div>;
```
маленькая поправка читабельности:)